options/posix/generic/sched-stubs.cpp

```diff
@@ -49,3 +54,17 @@ int clone(int (*)(void *), void *, int, void *, ...) {
 	__ensure(!"Not implemented");
 	__builtin_unreachable();
 }
+
+#ifdef __MLIBC_LINUX_OPTION
```

Comment:
This is probably better handled with an extra ``linux-sched`` file in the linux option.